src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -1,6 +1,10 @@
 // Adapted from freetype.h in order to avoid C-style casts.
 
 #define FT_LOAD_TARGET_ALT(x) (static_cast<FT_Int32>((x)&15) << 16)
+#define FT_IMAGE_TAG(value, _x1, _x2, _x3, _x4)                                                                                \
+    value                                                                                                                      \
+        = ((static_cast<unsigned long>(_x1) << 24) | (static_cast<unsigned long>(_x2) << 16)                                   \
+           | (static_cast<unsigned long>(_x3) << 8) | static_cast<unsigned long>(_x4))
```

Comment:
I didn't look this through, but because this is a macro, the callers might be sending things that aren't really `unsigned_long`, I think if we want to be safe, it should be a template instead

(Where is it coming from anyway? Because there are no red lines, so it seems like you're introducing this macro)

Reply:
This is a macro that is deep in the guts of the Freetype library. We don't even call it ourselves. But since GCC seems to make no distinction between our own headers and those of dependencies, I _have_ to define it. For this reason, I really can't change how it works.